.gitignore

```diff
@@ -34,4 +34,8 @@ out/
 /.nb-gradle/
 
 ### VS Code ###
+<<<<<<< HEAD
```

Comment:
it see u committed with  merge issues

Reply:
My .gitignore file is identical to main's. My merge conflicts were resolved in Github last time I checked which was last night in group?